target/linux/ramips/dts/F5D8235_V1.dts

```diff
@@ -68,15 +68,70 @@
 	gpio-leds {
 		compatible = "gpio-leds";
 
+		internet {
+			label = "f5d8235-v1:blue:internet";
+			gpios = <&gpio0 17 1>;
```

Comment:
Please include the dt-bindings/gpio/gpio.h header and use the GPIO_ACTIVE_LOW/GPIO_ACTIVE_HIGH macros. For reference: https://git.lede-project.org/a7de18718d4853e5387f51651573c79bc8a81169